python/setup.py.in

```diff
@@ -23,6 +23,21 @@ with open('@PADDLE_SOURCE_DIR@/python/requirements.txt') as f:
 if '${CMAKE_SYSTEM_PROCESSOR}' not in ['arm', 'armv7-a', 'aarch64']:
     setup_requires+=["opencv-python"]
 
+# the prefix is sys.prefix which should always be usr
+paddle_bin_dir = 'local/opt/paddle/bin'
```

Comment:
Thought `paddle` command is installed by `scripts=['${PADDLE_BINARY_DIR}/paddle/scripts/paddle']`, so it's installed to where python installed at. When running pip install without setting `--prefix` `paddle` command will try to find binaries relatively, whether in docker or not.

In @luotao1 's case, is it working if binaries are installed under `/home/luotao/.jumbo/opt/paddle/bin/`? If so, can using paddle_bin_dir = 'opt/paddle/bin'` solve all the cases?